_inc/client/search/index.jsx

```diff
@@ -0,0 +1,184 @@
+	isActivatingModule,
+	isDeactivatingModule,
+	getModule as _getModule,
+	getModules as _getModules
+} from 'state/modules';
+import { getSearchTerm } from 'state/search';
+import {
+	getSitePlan
+} from 'state/site';
+import ProStatus from 'pro-status';
+
+export const Page = ( {
+	toggleModule,
+	isModuleActivated,
+	isTogglingModule,
+	getModule,
+	getModules,
+	searchTerm,
+	unavailableInDevMode
+	} ) => {
+	let modules = getModules(),
+		moduleList = [
+			[
+				'scan',
+				__( 'Security Scanning' ),
+				__( 'Automatically scan your site for common threats and attacks.' ),
+				'security scan threat attacks pro' // Extra search terms @todo make translatable
+			],
+			[ 'akismet',
+				'Akismet',
+				__( 'Keep those spammers away!' ),
+				'https://akismet.com/jetpack/',
+				'spam security comments pro'
+			],
+			[
+				'backups',
+				__( 'Site Backups' ),
+				__( 'Keep your site backed up!' ),
+				'https://vaultpress.com/jetpack/',
+				'backup restore pro security'
+			]
+		],
+		cards;
+
+	forEach( modules, function( m ) {
+		'vaultpress' !== m.module ? moduleList.push( [
+			m.module,
+			getModule( m.module ).name,
+			getModule( m.module ).description,
+			getModule( m.module ).long_description,
+			getModule( m.module ).learn_more_button,
+			getModule( m.module ).search_terms,
+			getModule( m.module ).additional_search_queries,
+			getModule( m.module ).short_description,
+			getModule( m.module ).feature.toString()
+		] ) : '';
+	} );
+
+	cards = moduleList.map( ( element ) => {
+		let isPro = 'scan' === element[0] || 'akismet' === element[0] || 'backups' === element[0],
+			proProps = {},
+			unavailableDevMode = unavailableInDevMode( element[0] ),
+			toggle = unavailableDevMode ? __( 'Unavailable in Dev Mode' ) : (
+				<ModuleToggle
+					slug={ element[0] }
+					activated={ isModuleActivated( element[0] ) }
+					toggling={ isTogglingModule( element[0] ) }
+					toggleModule={ toggleModule }
+				/>
+			),
+			customClasses = unavailableDevMode ? 'devmode-disabled' : '';
+
+		if ( isPro ) {
+			proProps = {
+				module: element[0],
+				configure_url: 'backups' === element[0] || 'scan' === element[0] ?
+					'https://dashboard.vaultpress.com' :
+					Initial_State.adminUrl + 'admin.php?page=akismet-key-config'
+			};
+			toggle = <ProStatus proFeature={ element[0] } />;
+
+			// Add a "pro" button next to the header title
+			element[1] = <span>
+				{ element[1] }
+				<Button
+					compact={ true }
+					href="#professional"
+				>
+					{ __( 'Pro' ) }
+				</Button>
+			</span>
+		}
+
+		if ( 1 === element.length ) {
+			return ( <h1>{ element[0] }</h1> );
+		}
+
+		return (
+			<FoldableCard
+				key={ element[0] }
+				className={ customClasses }
+				header={ element[1] }
+				searchTerms={ element.toString().replace( /<(?:.|\n)*?>/gm, '' ) }
+				subheader={ element[2] }
+				summary={ toggle }
+				expandedSummary={ toggle }
+				clickableHeaderText={ true }
+			>
+				{
+					isModuleActivated ?
```

Comment:
looks like this should be a function call to `isModuleActivated` instead of a var reference